db/lib/types.js

```diff
@@ -1,4 +1,14 @@
 import { Type } from '@sinclair/typebox'
+
+const PayloadRetrievabilityState = {
+  NotQueried: 'PAYLOAD_CID_NOT_QUERIED_YET',
+  Unresolved: 'PAYLOAD_CID_UNRESOLVED',
+  Resolved: 'PAYLOAD_CID_RESOLVED',
+  TerminallyUnretrievable: 'PAYLOAD_CID_TERMINALLY_UNRETRIEVABLE'
+}
+
+const PayloadRetrievabilityStateType = Type.Enum(PayloadRetrievabilityState)
```

Comment:
```suggestion
const PayloadRetrievabilityStates = {
  NotQueried: 'PAYLOAD_CID_NOT_QUERIED_YET',
  Unresolved: 'PAYLOAD_CID_UNRESOLVED',
  Resolved: 'PAYLOAD_CID_RESOLVED',
  TerminallyUnretrievable: 'PAYLOAD_CID_TERMINALLY_UNRETRIEVABLE'
}

const PayloadRetrievabilityState = Type.Enum(PayloadRetrievabilityStates)
```

Does this make more sense? Payload Retrievability States lists all possible states. Payload Retrievability State is one of these states at a time.

We might want to wait for @bajtos with this one

Reply:
Your suggestion makes sense to me.